### DISCUSSION TITLE
Is Siimple still being maintained ? Active?

### DISCUSSION THREAD

**Discussion by martindufort** [jmjuanes/siimple#201] [General]
Hi, 

I just discovered siimple.xyz and love the simplicity of the project.
However the last Github update (V4.3.1) dates back to October 2022. 

Is this project still being maintained?

Thanks - Martin

**Comment by jmjuanes** [comment]
Hello!

Yes, it is still maintained. It's true that I have been quite busy with other projects, but at this moment the project is relatively stable. Also, I have some plans for the next version `v5`. 

If you have any questions or issues, please feel free to ask!

Thank you!

**Comment by martindufort** [comment]
About to integrate this framework into our web-based meeting management web site. 
Would love to get somekind of version `v5` roadmap published here.

Thanks

**Reply by jmjuanes** [reply]
First of all, thank you for reaching out and considering siimple for your project! I appreciate your interest.

After careful consideration, I have decided to maintain siimple at version 4 as it is a stable and reliable release. I believe stability is crucial for many projects, and I want to ensure that siimple continues to meet those needs. Rest assured, I will continue to address any potential bugs through the release of minor updates and patches.

Regarding the roadmap for version 5, I have shifted the ideas and improvements I had in mind for the next version of siimple into a new and independent project (I am working on it, next week I will make it publicly available). This allows me to explore new concepts without affecting the stability of siimple. If you are looking for stability and a well-established version, I recommend sticking with siimple version 4 for your current project.

On a related note, I also maintain another tiny CSS framework [lowcss](https://github.com/jmjuanes/lowcss). If you are interested in a utility-first CSS approach, you might find lowcss to be a suitable alternative.

Feel free to ask if you have any more questions or if there is anything else I can assist you with.

**Comment by martindufort** [comment]
> I have shifted the ideas and improvements I had in mind for the next version of siimple into a new and independent project (I am working on it, next week I will make it publicly available). 

That sounds really interesting. Would love to have a look and see if it integrates into our roadmap. 
Let me know when it's out in the open.

Thanks

**Comment by martindufort** [comment]
@jmjuanes Any updates on that new project?

**Reply by jmjuanes** [reply]
Hi there! Unfortunately, I haven't had as much time as I would like to dedicate to the project recently :disappointed:

However, I have uploaded the current progress to a public url, so you can take a look at what's been done so far: https://storage.googleapis.com/josemi-dev.appspot.com/hirocss/index.html (note that this is just a simple demo).

The new project is built on CSS variables, offering a flexible and customizable structure. Feel free to check out the current state, and I appreciate your patience and understanding as I continue to work on refining and expanding the library. If you have any specific questions or suggestions, I am all ears!

**Reply by martindufort** [reply]
I forgot to reply to this. Sorry for the delay. 
I think for our work, we will stick with Siimple as we have already integrated part of it. 

Thanks for this...
